readme.md

```diff
@@ -64,6 +64,9 @@ Use this as your HTML, load the script asynchronously, and let the JS progressiv
 </lite-youtube>
 ```
 
+## Libraries Using Other Frameworks
```

Comment:
Why do we want to put this one thing into its own section? I don't see a difference between this heading and "Other lite embeds" in terms of a reader. They are all alternative forms of this type of thing.

Reply:
Ok, I have reviewed both sections carefully and tried to combine them in a way that makes sense and offers the most information for users reading this section (so that they can make an informed decision on what they want to click on).

What do you think?